includes/posting.inc.php

```diff
@@ -1,1365 +1,1646 @@
 <?php
-if (!defined('IN_INDEX')) {
-	header('Location: ../index.php');
-	exit;
-}
-
-if (empty($_SESSION[$settings['session_prefix'].'user_id']) && $settings['captcha_posting'] > 0) {
-	require('modules/captcha/captcha.php');
-	$captcha = new Captcha();
-}
-
-// reorder categories if user has a personal category selection (selected categories first):
-if (isset($category_selection)) {
-	// put selected cegories in $reordered_categories:
-	foreach ($category_selection as $cat) {
-		$reordered_categories[$cat] = $categories[$cat];
+	if (!defined('IN_INDEX')) {
```

Comment:
Warum die erhöhte Einrückung? Ist da etwas schiefgelaufen?

Reply:
I used a online service to re-format the file.